src/components/blog/post/body.module.css

```diff
@@ -0,0 +1,358 @@
+.root {
+  font-family: "Acta", serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 28px;
+}
+
+/* Limit block elements w/ paddings */
+
+.root blockquote,
```

Comment:
Don't we have `sass` like nesting here? I think that would be prettier and easy to read.

Reply:
Is that a thing with CSS modules? :open_mouth: 

Just tested and seems it ain't. But if that's a possibility without adding junk I want it!

Reply:
Adding `postcss-nested` in #260 :heart: